lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -323,6 +325,18 @@ module InternalConstants
                  :default => nil,
                  :secret => true
 
+    # The URL of Stackdriver Logging API. Right now this only works with the
+    # gRPC path (use_grpc = true).
+    # The default value is: https://logging.googleapis.com.
+    # One common usage of this config is to provide a mocked / stubbed Logging
+    # API. e.g. http://localhost:52000.
```

Comment:
Careful -- your current code won't work with the `http://` prefix (it should, though).
Also, just wondering: do we really want to encode a port here?